scripts/workers/submission_worker.py

```diff
@@ -785,6 +795,8 @@ def main():
     queue = get_or_create_sqs_queue(queue_name)
     while True:
         for message in queue.receive_messages():
+            if message.body.get("is_static_dataset_code_upload_submission"):
+                continue
```

Comment:
@KhalidRmb Basically when `is_static_dataset_code_upload_submission` is true we do the creation of job on eks cluster and create submission file in `submission_input_file`. When this submission is done we add a message in this queue so, the submission would be evaluated here where the input file is not stored in submission.input_file but in submission.submission_input_file.

Reply:
Since we are using the same queue we are adding the check that the creation of job part is not considered. in case of evaluation stage `is_static_dataset_code_upload_submission` would be false.